src/utils/SVGIcons.ts

```diff
@@ -0,0 +1,24 @@
+import { $$, Dom } from './Dom';
+
+declare var require: (svgPath: string) => string;
+
+export class SVGIcons {
+  public static search = require('svg/search');
+  public static more = require('svg/more');
+  public static facetLoading = require('svg/facet-loading');
+  public static checkboxMoreValues = require('svg/checkbox-hook-exclusion-more');
+  public static arrowUp = require('svg/arrow-up');
+  public static arrowDown = require('svg/arrow-down');
+
+  private static parser = new DOMParser();
+
+  public static addClassToSVGInContainer(svgContainer: HTMLElement, classToAdd: string) {
```

Comment:
I don't want to manipulate SVG elements with the `Dom` class because they do not support the same methods as HTML elements, so some methods in the `Dom` class are not valid for SVG elements. I could add an interface that only exposes the correct methods when passing an svg element to the `Dom` class.